src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
+    public static final Map<String, Integer> fruitRepository = new HashMap<>();

```

Comment:
According to the encapsulation principle, it's a bad practice to make your class fields public or static. Consider making `fruitRepository` private and provide access to it through public methods.